Gopkg.toml

```diff
@@ -11,6 +11,10 @@
   name = "github.com/ugorji/go"
   revision = "ded73eae5db7e7a0ef6f55aace87a2873c5d2b74"
 
-[[override]]
+[[constraint]]
   name = "github.com/spf13/cobra"
-  revision = "d5bde60e022436203e377c47806e001418c16af3"
\ No newline at end of file
+  version = "v0.0.5"
+
+[[override]]
+  name = "github.com/sirupsen/logrus"
+  revision = "07a84ee7412e7a28663d92930a1d46f81b124ee1"
```

Comment:
Could this be a version rather than a commit e.g. 1.4.2?

Reply:
Great question, unfortunately they have not yet cut a version that includes the PadLevelText option to the TextFormatter . This has been merged to the logrus master branch just not out in a release yet. Once the next release is out we can switch to that.

Reply:
Ah, okay. In that case using the revision is fine.